Translate Hello World (#57)
reactjs/pt-BR.reactjs.org: content/docs/hello-world.md

);
```

Isso mostra um header dizendo "Hello, world!" na página.

[](codepen://hello-world)

Clique no link acima para abrir um editor online. Sinta-se livre para fazer algumas mudanças e ver como elas afetam a saída. A maioria das páginas neste guia terão exemplos editáveis como esse.

> ## Como Ler Esse Guia {#como-ler-esse-guia}

Reviewer: ```suggestion
## Como Ler Esse Guia {#how-to-read-this-guide}
```

> Nesse guia, vamos examinar as fundações de aplicações React: elementos e componentes. Depois que você dominá-los, poderá criar aplicações complexas a partir de partes pequenas e reutilizáveis.

Reviewer 2: ```suggestion
Nesse guia, vamos examinar as fundações de aplicações React: elementos e componentes. Depois que você tiver o domínio, poderá criar aplicações complexas a partir de partes pequenas e reutilizáveis.
```

>Dica
>
>Esse guia é destinado a pessoas que preferem **aprender conceitos passo a passo**. Se você prefere aprender fazendo, confira nosso [tutorial prático](/tutorial/tutorial.html). Você pode acabar descobrindo que esse guia e o tutorial se complementam.

> Esse é o primeiro capítulo em um guia passo-a-passo sobre os principais conceitos do React. Uma lista de todos os capítulos pode ser encontrada na barra de navegação lateral. Se você estiver lendo em um dispositivo móvel, pode acessar a navegação pressionando o botão no canto inferior direito da sua tela.

Reviewer 2: ```suggestion
Esse é o primeiro capítulo de um guia passo-a-passo sobre os principais conceitos do React. Uma lista de todos os capítulos pode ser encontrada na barra de navegação lateral. Se você estiver lendo em um dispositivo móvel, pode acessar a navegação pressionando o botão no canto inferior direito da sua tela.
```

> Todo capítulo nesse guia contrói em cima do conhecimento introduzido em capítulos anteriores. **Você pode aprender muito de React lendo os capítulos do guia de "Conceitos principais" na ordem em que eles aparecem na barra lateral.** Por exemplo, [Introduzindo JSX”](/docs/introducing-jsx.html) é o próximo capítulo depois desse.

Reviewer: ```suggestion
Todo capítulo nesse guia contrói em cima do conhecimento introduzido em capítulos anteriores. **Você pode aprender muito do React lendo os capítulos do guia de "Conceitos principais" na ordem em que eles aparecem na barra lateral.** Por exemplo, [Introduzindo JSX”](/docs/introducing-jsx.html) é o próximo capítulo depois desse.
```

> Todo capítulo nesse guia contrói em cima do conhecimento introduzido em capítulos anteriores. **Você pode aprender muito de React lendo os capítulos do guia de "Conceitos principais" na ordem em que eles aparecem na barra lateral.** Por exemplo, [Introduzindo JSX”](/docs/introducing-jsx.html) é o próximo capítulo depois desse.

Reviewer 2: ```suggestion
Todo capítulo nesse guia se desenvolve em cima do conhecimento introduzido em capítulos anteriores. **Você pode aprender muito do React lendo os capítulos do guia de "Conceitos principais" na ordem em que eles aparecem na barra lateral.** Por exemplo, [Introduzindo JSX”](/docs/introducing-jsx.html) é o próximo capítulo após esse.
```

> ## Suposições de nível de conhecimento {#suposicoes-de-nivel-de-conhecimento}

Reviewer: ```suggestion
## Suposições de nível de conhecimento {#knowledge-level-assumptions}
```

> React é uma biblioteca Javascript, então assumiremos que você tem entendimento básico da linguágem, **Se não se sentir confiante, nós recomendamos: [seguir um tutorial de javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para checar o seu nível de conhecimento e garantir que você poderá acompanhar esse guia sem se perder. Isso pode levar entre 30 minutos e uma hora, mas você não sentirá como se estivesse aprendendo React e javaScript o mesmo tempo.

Reviewer: ```suggestion
React é uma biblioteca JavaScript, então assumiremos que você possui um entendimento básico da linguagem. **Se não se sentir confiante, nós recomendamos: [seguir um tutorial de javascript](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para checar o seu nível de conhecimento e garantir que você poderá acompanhar esse guia sem se perder. Isso pode levar entre 30 minutos e uma hora, mas você não sentirá como se estivesse aprendendo React e JavaScript ao mesmo tempo.
```

> React é uma biblioteca Javascript, então assumiremos que você tem entendimento básico da linguágem, **Se não se sentir confiante, nós recomendamos: [seguir um tutorial de javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para checar o seu nível de conhecimento e garantir que você poderá acompanhar esse guia sem se perder. Isso pode levar entre 30 minutos e uma hora, mas você não sentirá como se estivesse aprendendo React e javaScript o mesmo tempo.

Reviewer 2: ```suggestion
React é uma biblioteca JavaScript. Então assumiremos que você já possui um entendimento básico da linguagem. **Se não se sentir confiante, nós recomendamos: [seguir um tutorial de javascript](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para checar o seu nível de conhecimento e garantir que você acompanhará esse guia sem se perder. Isso pode levar entre 30 minutos e uma hora. Mas você não sentirá como se estivesse aprendendo React e JavaScript ao mesmo tempo.
```

>Nota
>

> >Esse guia ocasionalmente usa partes da nova sintáxe do JavaScript nos exemplos. Se você não tem trabalhado com Javascript nos ultimos anos, [esses três pontos:](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) devem te ajudar em boa parte do caminho.

Reviewer: ```suggestion
>Esse guia ocasionalmente usa partes da nova sintaxe do JavaScript nos exemplos. Se você não tem trabalhado com JavaScript nos últimos anos, [esses três pontos](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) devem te ajudar em boa parte do caminho.
```

> >Esse guia ocasionalmente usa partes da nova sintáxe do JavaScript nos exemplos. Se você não tem trabalhado com Javascript nos ultimos anos, [esses três pontos:](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) devem te ajudar em boa parte do caminho.

Reviewer 2: ```suggestion
Esse guia geralmente usa partes da nova sintaxe do JavaScript nos exemplos. Se você não tem trabalhado com Javascript nos últimos anos, [esses três pontos:](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) lhe ajudará em boa parte do caminho.
```

> ## Vamos começar! {#vamos-comecar}

Reviewer: ```suggestion
## Vamos começar! {#lets-get-started}
```

## Let's Get Started! {#lets-get-started}

> Continue rolando para baixo, e você encontrará o link para o [próximo capítulo desse guia](/docs/introducing-jsx.html) bem antes do rodapé.

Reviewer: ```suggestion
Continue rolando para baixo, e você encontrará o link para o [próximo capítulo desse guia](/docs/introducing-jsx.html) logo acima do rodapé.
```
